randovania/layout/base/trick_level_configuration.py

```diff
@@ -39,7 +39,7 @@ def bit_pack_encode(self, metadata) -> Iterator[Tuple[int, int]]:
         encodable_levels = list(LayoutTrickLevel)
         encodable_levels.remove(LayoutTrickLevel.DISABLED)
 
-        for trick in sorted(_all_tricks(resource_database)):
+        for trick in _all_tricks(resource_database):
```

Comment:
Any reason it doesn't sort anymore? The sort was there because this code is highly dependant on the order and having a sort makes to not care so much whatever happen on the other end